src/components/Details/Details.tsx

```diff
@@ -23,23 +24,22 @@ interface SelectedMarketProps {
   id: string;
 }
 
-export const Details: React.FC<SelectedMarketProps> = ({
-  selectedMarket,
-  id
+export const Details: React.FC<SelectedMarketProps> = ({ 
+  selectedMarket, 
+  id 
 }) => {
-  const openSeason = `Season: ${selectedMarket?.schedule[0].season}`;
-
-  const daysAndTimes = selectedMarket?.schedule.map((sched, index) => {
-    return (
-      <div key={index}>
-        <ul className='day-list'>
-          <li>
-            {sched.dayOfWeek} {sched.time}
-          </li>
-        </ul>
-      </div>
-    );
-  });
+  let [error, setError] = useState("");
+  let [ openSeason, setOpenSeason ] = useState("")
+  let [ daysAndTimes, setDaysAndTimes ] = useState("")
+ 
+  useEffect(() => {
+    if (!selectedMarket?.schedule.length) {
+      setError("Uh oh! Market not found! Please try again.")
```

Comment:
My take away from a market not having a season in the API wasn't that the market wasn't found, but that a schedule wasn't registered in their system. My preference would be to render something that said as much. Ex: "No schedule available"